### DISCUSSION TITLE
Can AutoConnect co-exist with ESPAsyncWebServer

### DISCUSSION THREAD

**Discussion by Charles-Goodwin** [Hieromon/AutoConnect#390] [Q&A]
I have a project that uses ESPAsyncWebServer to provide a web page that controls my lamp.
The code is here: [https://github.com/Charles-Goodwin/Lamp](url)
Currently the wifi credentials are hard coded and I love what you have achieved with AutoConnect.
I've tried to integrate your code library into my project but it's not straight forward and an error is thrown when I try to bind the Async Server to the Portal

Is there a way for AutoConnect and the ESPAsyncWebServer libraries to co-exist?

**Comment by Hieromon** [comment]
Refer to https://github.com/Hieromon/AutoConnect/discussions/351
